### ISSUE TITLE
When bypassing a a group node containing a Apply Advanced ControlNet 🛂🅐🅒🅝, the positive and negative conditioning get switched.

### ISSUE THREAD

**Issue by dchatel** [Kosinkadink/ComfyUI-Advanced-ControlNet#70]
This is quite simple. The steps to reproduce simply consist in creating a group node with Apply Advanced ControlNet 🛂🅐🅒🅝, then bypassing that group node.
When bypassed, the positive and negative conditioning will switch places: the positive get linked to the negative and vice versa. which leads to... pretty creepy but sometimes hilarious stuff.

This is what we get when bypassing Apply Advanced ControlNet 🛂🅐🅒🅝 when **ungrouped**, so this is also what is expected when bypassing a grouped node: https://ibb.co/WWw3ZBT
But this is what we get when bypassing a grouped node containing Apply Advanced ControlNet 🛂🅐🅒🅝 (Warning: it hurts the eyes a little bit): https://ibb.co/x569F61

**Comment by Kosinkadink** [comment]
I have no custom javascript at all in my node, so my hunch is this may be a bug with how ComfyUI handles bypassing grouped nodes. Can you see if the issue also happens if you use the vanilla Apply ControlNet (Advanced) node that's built into comfy?

**Comment by dchatel** [comment]
Yes, it happens even with the vanilla apply controlnet (advanced). I've forgot to update you on the matter, my bad.

**Comment by Kosinkadink** [comment]
Gotcha, you should open up an issue on ComfyUI repo - there is nothing I can do about this on my end.
